packages/devel/boost/package.mk

```diff
@@ -1,77 +1,77 @@
-#      This file is part of OpenELEC - http://www.openelec.tv
+#      This file is part of LibreELEC - https://libreelec.tv
+#      Copyright (C) 2016-present Team LibreELEC
 #      Copyright (C) 2009-2016 Stephan Raue (stephan@openelec.tv)
 #
-#  OpenELEC is free software: you can redistribute it and/or modify
+#  LibreELEC is free software: you can redistribute it and/or modify
 #  it under the terms of the GNU General Public License as published by
 #  the Free Software Foundation, either version 2 of the License, or
 #  (at your option) any later version.
 #
-#  OpenELEC is distributed in the hope that it will be useful,
+#  LibreELEC is distributed in the hope that it will be useful,
 #  but WITHOUT ANY WARRANTY; without even the implied warranty of
 #  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 #  GNU General Public License for more details.
 #
 #  You should have received a copy of the GNU General Public License
-#  along with OpenELEC.  If not, see <http://www.gnu.org/licenses/>.
+#  along with LibreELEC.  If not, see <http://www.gnu.org/licenses/>.
 ################################################################################
 
 PKG_NAME="boost"
-PKG_VERSION="1_61_0"
-PKG_SHA256="a547bd06c2fd9a71ba1d169d9cf0339da7ebf4753849a8f7d6fdb8feee99b640"
-PKG_ARCH="any"
+PKG_VERSION="1_65_1"
+PKG_SHA256="9807a5d16566c57fd74fb522764e0b134a8bbe6b6e8967b83afefd30dcd3be81"
 PKG_LICENSE="OSS"
 PKG_SITE="http://www.boost.org/"
-PKG_URL="$SOURCEFORGE_SRC/boost/boost/1.61.0/${PKG_NAME}_${PKG_VERSION}.tar.bz2"
+PKG_URL="$SOURCEFORGE_SRC/boost/boost/1.65.1/${PKG_NAME}_${PKG_VERSION}.tar.bz2"
 PKG_SOURCE_DIR="${PKG_NAME}_${PKG_VERSION}"
 PKG_DEPENDS_HOST=""
-PKG_DEPENDS_TARGET="toolchain boost:host Python:host zlib bzip2"
-PKG_SECTION="devel"
-PKG_SHORTDESC="boost: Peer-reviewed STL style libraries for C++"
-PKG_LONGDESC="Boost provides free peer-reviewed portable C++ source libraries. The emphasis is on libraries which work well with the C++ Standard Library. One goal is to establish existing practice and provide reference implementations so that the Boost libraries are suitable for eventual standardization. Some of the libraries have already been proposed for inclusion in the C++ Standards Committee's upcoming C++ Standard Library Technical Report."
-
-PKG_IS_ADDON="no"
+PKG_DEPENDS_TARGET="toolchain boost:host Python zlib bzip2"
+PKG_LONGDESC="boost: Peer-reviewed STL style libraries for C++"
 PKG_AUTORECONF="no"
 
 make_host() {
   cd tools/build/src/engine
-    sh build.sh
+  sh build.sh
 }
 
 makeinstall_host() {
   mkdir -p $TOOLCHAIN/bin
-    cp bin.*/bjam $TOOLCHAIN/bin
+  cp bin.*/bjam $TOOLCHAIN/bin
 }
 
 pre_configure_target() {
-  export CFLAGS="$CFLAGS -fPIC"
-  export CXXFLAGS="$CXXFLAGS -fPIC"
-  export LDFLAGS="$LDFLAGS -fPIC"
+  export CFLAGS="$CFLAGS -I$SYSROOT_PREFIX/usr/include/python2.7 -fPIC"
+  export CXXFLAGS="$CXXFLAGS -I$SYSROOT_PREFIX/usr/include/python2.7 -fPIC"
```

Comment:
I'm not sure this is desirable as we have already started to experiment with python 3 kodi builds. The transition is still a way off.. but this might cause issues @MilhouseVH ??

Reply:
This is required to build the libboost_python library and can easily be changed for Python 3.

Reply:
No issues from me. While implementing Python3 I did add a build function that returned the python version so we didn't hard-code `python2.7`, `python3.6`, etc. but that's fairly trivial in the scheme of things and something that can be fixed up in future if we decide to go that route. It's actually something that could be introduced ahead of Python3 itself.